integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/GroupsResource.java

```diff
@@ -150,4 +150,11 @@ List<GroupRepresentation> groups(@QueryParam("search") String search,
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     List<GroupRepresentation> query(@QueryParam("q") String searchQuery, @QueryParam("populateHierarchy") boolean populateHierarchy);
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    List<GroupRepresentation> query(@QueryParam("q") String searchQuery,
```

Comment:
Doesn't make sense to also include `search` and `exact` parameters?

Reply:
I was more thinking about providing a method that you can use both as we are missing it. But not hard on that.